week-3/Homework/mandatory/1-oxygen-levels.js

```diff
@@ -27,32 +26,36 @@ console.log(newdessert); // returns ice cream
 /* + + + + + + + */
 /* Now try the exercise */
 
-function safeLevels() {
+function safeLevels(arr) {
+  let output = arr.find(function (item) {
+    return item > "19.5%" && item < "23.5%";
```

Comment:
there is a bit of confusion here. Your test may pass but this is not correct.
try to check `console.log("3%">"22%")` 
The % sign makes it a string that is not castable to a number. What's happening now when you put ">" between them is string comparison and this happen letter by letter, so 2>0, and 3 > 1000.
Think about a way to cast it to number first